apps/storybook/src/Slider.stories.tsx

```diff
@@ -168,7 +167,7 @@ CustomTickNoTooltip.decorators = [
       <Story />
     </div>
   ),
-];
+] satisfies StoryDecorator[];
```

Comment:
There are other similar uses of `ComponentXYZ.decorators = ...` that likely also need `satisfies StoryDecorator[]` to fix the type errors. Example places: ButtonGroup L194, Carousel L179, etc.

You can do these changes in this PR, or I had planned going through all stories and fixing type errors. So I can do it then too.

Reply:
fixed all decorators https://github.com/iTwin/iTwinUI/pull/1669/commits/46c2c6b8047f10ecee307d2aacc430db3e7a9655